Mlem/App/Views/Root/Tabs/Settings/InteractionBarEditor/InteractionBarWidgetPickerView.swift

```diff
@@ -32,20 +33,23 @@ struct InteractionBarWidgetPickerView<Configuration: InteractionBarConfiguration
                 }
             }
         }
+        .toolbar {
+            ToolbarItem(placement: .topBarTrailing) {
+                CloseButtonView {
+                    dismiss()
+                }
```

Comment:
This has default behavior if no callback is provided
```suggestion
                CloseButtonView()
```